src/Pages/Main/OptionSlide.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect, useRef } from "react";
+import styled from "styled-components";
+import Slide from "./Slide";
+
+export default function OptionSlide({ img1, img2, img3, count, height }) {
+  const TOTAL_SLIDES = count;
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const slideRef = useRef(null);
+
+  const nextSlide = () => {
+    if (currentSlide >= TOTAL_SLIDES) {
+      setCurrentSlide(0);
+    } else {
+      setCurrentSlide(currentSlide + 1);
+    }
+  };
```

Comment:
아래와 같이 길어지는 조건에는 변수 사용, 그리고 삼항 연산자 사용해서 코드를 수정할 수 있겠네요!
```suggestion
  const nextSlide = () => {
    const isMaximum = currentSlide >= TOTAL_SLIDES;
    setCurrentSlide(isMaximum ? 0 : currentSlide + 1);
  };
```

Reply:
와 👏